net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -36,15 +38,32 @@ MM_UNREACHABLE=""
 command -v ip6tables > /dev/null
 NO_IPV6=$?
 
+SCRIPTNAME="$(basename "$0")"
+LOG()
+{
+	local facility=$1; shift
+	# in development, we want to show 'debug' level logs
+	# when this release is out of beta, the comment in the line below
+	# should be removed
+	# [ "$facility" = "debug" ] && continue
+	logger -t "$SCRIPTNAME[$$]" -p $facility "$*"
+}
+
+
+mwan3_push_update()
```

Comment:
i do not know this syntax. Can you please tell me exactly what you're doing here.
current and update?

Reply:
Sure. It's a poor-man's stringbuilder. Rather than making a bunch of separate `iptables` calls, it is faster to create everything in an update variable and then do one `iptables restore` call per function.  The syntax looks a bit weird here because I need to put in a new line after each update, and (as far as I can tell) `ash` doesn't allow escaped new lines in string literals, so there is an actual new line there. I can add a comment there to make things clearer.